model_hub/model_hub/mmdetection/_trial.py

```diff
@@ -63,14 +64,10 @@ def __init__(self, context: det_torch.PyTorchTrialContext) -> None:
 
         # Build model and make sure it's compatible with horovod.
         self.model = mmdet.models.build_detector(self.cfg.model)
-        # We need to monkey patch the reduce_mean function from mmdet.core.utils so that it uses
```

Comment:
we should probably convert everything to work with native pytorch dtrain only since we are going to remove the monkey patches here.

Reply:
we should remove the monkey patch in utils.py and also update the entrypoint in all the experiment configs to go through torch_distributed

Reply:
done

Reply:
done